.changeset/dull-tables-talk.md

```diff
@@ -0,0 +1,14 @@
+---
+'eslint-config-seek': major
```

Comment:
I don't have a strong opinion on how we should semver this. The new recommended rule would fail on many existing projects but it is autofixable.

Reply:
imo I think major is appropriate given it is breaking despite being autofixable.